test/runtests.py

```diff
@@ -469,17 +469,28 @@ def timeout_func(timeout_data):
             timeout_data[1] = True
         timeout = test.get('timeout', args.timeout) # test override or default
         timer = Timer(timeout, timeout_func, [timeout_data])
+        force_match_baseline = False
```

Comment:
>force_match_baseline [](start = 8, length = 20)

I confused `force_match_baseline` as "force to do baseline match", surprised as that was the default behavior. Would it be more clear to name this sth. like "skip_baseline_match"?